extensions/ql-vscode/package.json

```
@@ -674,12 +678,12 @@
         {
           "command": "codeQLQueryHistory.compareWith",
           "group": "9_qlCommands",
-          "when": "view == codeQLQueryHistory"
+          "when": "view == codeQLQueryHistory && (viewItem == rawResultsItem || viewItem == interpretedResultsItem)"
```

"Compare With" wasn't visible at all (ditto for "Show Query Log" and "View DIL" below). This works locally: 
```suggestion
          "when": "view == codeQLQueryHistory && viewItem == rawResultsItem || view == codeQLQueryHistory && viewItem == interpretedResultsItem"
```

Equivalent, but VS Code just doesn't seem to support parentheses in `when` clauses 🙄